tin-drink/src/components/content/drinkList/SingleDrink.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './SingleDrink.css';
+
+class SingleDrink extends React.Component {
+
+    showDrink = () => {
+        console.log('show drink')
+    }
+
+    
+    render () {
+        console.log(this.props);
+        return(
+            <div id="single-drink" class="single-drink">
+                <div id="drink-photo" class="drink-photo">
+                    <img src={`http://localhost:8080/images/${this.props.drink.photoPath}`} onClick={(e) => this.showDrink()}/>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.